src/schema/schema_create.c

```diff
@@ -159,7 +159,11 @@ __create_file(WT_SESSION_IMPL *session, const char *uri, bool exclusive, const c
             WT_ERR(__wt_config_collapse(session, filecfg, &fileconf));
             WT_ERR(__wt_metadata_insert(session, uri, fileconf));
         } else {
-            /* TO-DO: WT-6691 */
+            /*
+             * In the repair case we need to use the import API call. This will read the data file's
```

Comment:
Is there a reason you want to use an API call to do the import repair?

Reply:
As discussed, I've removed the `import` option from the API and renamed the functionality to `__wt_import_repair`.